src/js/apps/patients/patient/patient_app.js

```
@@ -61,39 +78,38 @@
-    this.listenToOnce(actionApp, {
-      'start'(options, action) {
-        this.editActionList(action);
-      },
-      'fail'() {
-        this.startCurrent('dashboard');
-      },
-    });
-
-    this.startChildApp('action', { actionId, patientId });
+  onStop() {
+    delete this._list;
```

Biome

[error] Avoid the delete operator which can impact performance.

Unsafe fix: Use an undefined assignment instead.


(lint/performance/noDelete)